src/lib/autocomplete/autocomplete-trigger.ts

```diff
@@ -477,7 +477,12 @@ export class MatAutocompleteTrigger implements ControlValueAccessor, OnDestroy {
    * stream every time the option list changes.
    */
   private _subscribeToClosingActions(): Subscription {
-    const firstStable = this._zone.onStable.asObservable().pipe(take(1));
+    const firstStable = this._zone.onStable.asObservable().pipe(
+      // Defer emitting to the stream until the next tick. Without this,
+      // a propagating click event may close the panel immediately.
```

Comment:
Changes like these are difficult to merge in, because they'll break people's tests. Couldn't you work around this issue in your own code using `event.stopPropagation()`?

Reply:
@crisbeto  Sure...should the issue be closed or are you still looking for a fix?

Reply:
I'm open towards other solutions that don't involve adding a timeout.

Reply:
got it. will look into it. Thanks for the feedback!